docs/methods.rst

```diff
@@ -39,17 +39,21 @@ update
 
 Arguments:
 
+* date (String|Date|Array, optional)
 * date (String|Date, optional)
```

Comment:
@Azaret I think that you want to delete this line and leave `* date (String|Date|Array, optional)`.